server/src/main/java/org/hyperskill/community/flashcards/config/MongoConfiguration.java

```diff
@@ -15,10 +14,4 @@ public class MongoConfiguration {
     public MongoTemplate mongoTemplate() {
         return new MongoTemplate(MongoClients.create(), "cards");
     }
-
-    @Bean
-    @Qualifier("example")
-    public MongoTemplate exampleMongoTemplate() {
-        return new MongoTemplate(MongoClients.create(), "example");
-    }
 }
```

Comment:
so now you work on cards database?

Reply:
Yes, it populates the development `cards` database with 2 users and 12 flashcards of different types, all put in the `example` collection which belongs to the `test1@test.com` user. This is to facilitate development of endpoints and other components based on the actual database.